packages/types/api/providers.d.ts

```diff
@@ -0,0 +1,32 @@
+export interface ProviderRecord {
```

Comment:
Can we name this `DynamoProviderRecord` for clarity?

Reply:
See my comment above - I'm fine with that, but if we continue to use it, I'm going to want to refactor it when we eliminate dynamo.

Reply:
If we do that we should also update the execution type in this package.

Reply:
This is outdated via our discussion on 13 Nov. c591c12ea